ui/src/map/tools/measurement-tool.tsx

```diff
@@ -165,4 +169,17 @@ export const measurementTool: MapTool = {
             map.removeOverlay(tooltip);
         };
     },
+
+    component: ({ isActive, setActiveTool }) => {
+        return (
+            <li
+                onClick={() => setActiveTool(measurementTool)}
+                className={createClassName(
+                    styles['map__map-tool'],
+                    isActive && styles['map__map-tool--active'],
+                )}>
+                <Icons.Measure color={IconColor.INHERIT} />
+            </li>
```

Comment:
Note to self: Tämä DOM-rakenne toistuu about kaikissa karttatooleissa, voisi yhdistää